cms/templates/product_page.html

```diff
@@ -10,7 +11,72 @@
 {% endblock %}
 
 {% block content %}
-  <div class="course-catalog">
-    <h1>{{ page.title }}</h1>
-  </div>
+    <div id="wrapper">
+      <section class="hero-block">
+        <div class="container">
+          <div class="content-row">
+            <div class="content-col">
+              <div class="text">
+                <h1>{{ page.title }}</h1>
+                <p>{{ page.description |richtext }}</p>
+                <a href="#" class="btn btn-default">Enroll now</a>
+              </div>
+            </div>
+            <div class="content-col video">
+              <div class="video-holder">
+                <img src="{% static 'images/video.png' %}" alt="Video Holder">
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section class="info-block">
+        <div class="container">
+          <div class="stats-row">
+            <div class="stat-col">
+              <span class="title">START DATE</span>
+              <div class="text">
+                <strong>{{ page.product.first_unexpired_run.start_date|date:"F j, Y" }}</strong>
+              </div>
+            </div>
+            <div class="stat-col small">
+              <span class="title">LENGTH</span>
+              <div class="text">
+                <strong>{{ page.length }}</strong>
+              </div>
+            </div>
+            <div class="stat-col">
+              <span class="title">EFFORT</span>
+              <div class="text">
+                <strong>{{ page.effort }}</strong>
+              </div>
+            </div>
+            <div class="stat-col small">
+              <span class="title">PRICE</span>
+              <div class="text">
+                <strong>Varies</strong>
```

Comment:
This text should be managed in the CMS

Reply:
Thanks for answering this https://github.com/mitodl/mitxonline/pull/45#discussion_r680813560